src/routing/pageRoutes.tsx

```diff
@@ -90,7 +90,6 @@ const RouteContent = ({ title, description, jsx }: BasePage) => (
 
       <title>{title}</title>
       <meta name="description" content={description} />
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Removing this here because it turned out to already be present in `index.html` file.